window_installer/demo/crm_lead.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="crm_lead_1" model="crm.lead" forcecreate="1">
+    <field name="name">Installation of chassis in a proch </field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="partner_id" ref="res_partner_7"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead2"/>
+    <field name="date_open">2024-07-18 06:15:43</field>
+    <field name="description"><![CDATA[<p ></p><p>I wanted to contact you to get a quotation for the construction&nbsp;of a porch with mate chassis.</p><p>Best regards,</p><p>Constant Custinne</p>
+            
+        ]]></field>
+    <field name="expected_revenue">15000.0</field>
+  </record>
+  <record id="crm_lead_2" model="crm.lead" forcecreate="1">
+    <field name="name">Installation of 6 chassis for a room</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="partner_id" ref="res_partner_20"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead3"/>
+    <field name="country_id" ref="base.be"/>
+    <field name="probability">64.71</field>
+    <field name="description"><![CDATA[<p> Dear Mister Admin,</p><p>I'm reaching to get a quotation for the installation&nbsp;of these chassis:</p><p>2 frames of&nbsp;</p><ul><li>Height: 2 meters</li><li>Length: 1&nbsp;meters</li></ul><p>4 frame of :<br></p><ul><li>Height 1.4m</li><li>Length 60 cm</li></ul><p>Best regards,</p><p>Kamiel Van De Kust</p>
+            
+        ]]></field>
+    <field name="expected_revenue">5000.0</field>
+  </record>
+  <record id="crm_lead_3" model="crm.lead" forcecreate="1">
+    <field name="name">Information for installation</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead1"/>
+    <field name="probability">91.67</field>
+    <field name="won_status">pending</field>
+    <field name="date_open">2024-07-18 06:15:43</field>
+    <field name="description"><![CDATA[<p>Hello,</p><p>What are the charges per hour?</p><p>Thank you</p>
+            
+        ]]></field>
+  </record>
+  <record id="crm_lead_9" model="crm.lead">
+    <field name="name">Jackson Architects's opportunity</field>
+    <field name="partner_id" ref="res_partner_9"/>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead4"/>
+    <field name="date_closed">2024-07-18 07:08:09</field>
+    <field name="priority">2</field>
```

Comment:
```suggestion
```

Reply:
why?

Reply:
It's of no use 
Is it ?

Reply:
some lead can be more important than other..